platform-designs/design-architecture/README.md

```diff
@@ -12,7 +12,9 @@ IA focuses on two elements: **where a user is** and **where to find information
 The diagram reads as a hierarchy from top to bottom with each layer starting with a "parent" element that has corresponding "child" elements. 
 
 ### IA Diagram
-![AutSPACEs IA Diagram 15-11-2021](https://github.com/alan-turing-institute/AutisticaCitizenScience/blob/master/platform-designs/design-architecture/autspaces-ia-diagram-15-11-2021.md)
+* Updated 21-02-2022: [Information Architecture Diagram](https://viewer.diagrams.net/?tags=%7B%7D&highlight=0000ff&edit=_blank&layers=1&nav=1&page-id=XndytNqcHWh8WrCRgyDL&title=IA_AutSPACEs.drawio#Uhttps%3A%2F%2Fdrive.google.com%2Fuc%3Fid%3D1THiziivGBr89J5x9oJahXuHtL7iib8J1%26export%3Ddownload)
```

Comment:
Do we have an information capture about what was changed and why (e.g. we can write a sentence on line 15 about the changes).